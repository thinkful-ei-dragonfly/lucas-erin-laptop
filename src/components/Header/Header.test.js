import React from 'react';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json'
import Header from './Header'
configure({ adapter: new Adapter() });

  describe('Header', () => {
    it('renders with enzyme', () => {
      const wrapper = shallow(<Header />);
      expect(toJson(wrapper)).toMatchSnapshot();
    })
  })
