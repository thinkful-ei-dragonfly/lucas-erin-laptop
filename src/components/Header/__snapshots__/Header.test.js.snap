// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders with enzyme 1`] = `
<header>
  <h1>
    ELF Computing
  </h1>
  <h3>
    Laptops
  </h3>
  <h5>
    Customize your laptop
  </h5>
</header>
`;
